pkg/foundation/metrics/measure/measure.go

```diff
@@ -28,6 +28,16 @@ var (
 	PipelinesGauge = metrics.NewLabeledGauge("conduit_pipelines",
 		"Number of pipelines by status.",
 		[]string{"status"})
+	PipelineStatusGauge = metrics.NewLabeledGauge(
+		"conduit_pipeline_status",
+		"Pipeline statuses.",
+		[]string{"pipeline_name", "status"},
+	)
+	PipelineRecoveringCount = metrics.NewLabeledCounter(
+		"pipeline_recovering_count",
+		"Number of times pipelines have been recovering (by by pipeline name)",
```

Comment:
This would refer to a specific pipeline -by name- (and not pipelines, right?). Also fixing a double `by`

```suggestion
		"Number of times a pipeline have been recovering (by pipeline name)",
```